hw/ip/flash_ctrl/flash_ctrl.core

```diff
@@ -32,6 +32,7 @@ filesets:
       - rtl/flash_ctrl_arb.sv
       - rtl/flash_ctrl_info_cfg.sv
       - rtl/flash_ctrl_lcmgr.sv
+      - rtl/flash_ctrl_region_cfg.sv
```

Comment:
Hey @tjaychen could you take a look if this is really missing when you have time?
I wonder how DV passed without this file listed.
Did I use a wrong core file?

Reply:
wow what the heck, how did this never break before. Let me double check.

Reply:
ah, okay this is because of the stupid templated ip again.
The missing file is actually under [this core file](https://github.com/lowRISC/opentitan/blob/master/hw/ip/flash_ctrl/flash_ctrl.core#L24)

And block level DV actually overrides to use the [top flag](https://github.com/lowRISC/opentitan/blob/master/hw/ip/flash_ctrl/dv/flash_ctrl_sim_cfg.hjson#L40) 

So we can't actually fix the way this PR is showing, we should be qualifying with a flag instead. 
Let me do that.  But for fpv can you also override with the top flag?

Long term I need to move this to ip_templated / ip_autogen, then this won't be a problem anymore.

Reply:
@tjaychen yes! In FPV I can override to use the top flag :) Thanks for letting me know!
-> So i guess you do not need to make any change on design side as well right?